.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ jobs:
   integration_tests:
```

Comment:
Not sure what the impact is but the tag still references 12:     
`tag: 12-buster-browsers`